specs/refactoring_of_node_monitoring_flows_into_node_agent.adoc

```diff
@@ -0,0 +1,181 @@
+// vim: tw=79
+
+= Refactoring Of Node Monitoring Flows Into Node-Agent
+
+== Introduction
+
+This specification deals with detailing the work involved in merging the
+node-monitoring and node-agent services.
+
+Currently the responsibilities of node-monitoring are as follows:
+
+* It packages the collectd custom plugins
```

Comment:
I remember once we talked about possibilities of these collectd plugins to gluster org itsefl. May be for future.